### PR TITLE
Update Citus link to the github.com repo

### PR DESCRIPTION
**Pull request by CyberDem0n**
Per suggestion from @clairegiordano

### CODE REVIEW

**Comment by hughcapet** [comment]
👍

**Review by clairegiordano — COMMENTED** [review]
Minor suggestion re clarifying the word "support"

File: README.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ We call Patroni a "template" because it is far from being a one-size-fits-all or
 
 Currently supported PostgreSQL versions: 9.3 to 15.
 
-**Note to Citus users**: Starting from 3.0 Patroni nicely integrates with `Citus <https://www.citusdata.com>`__. Please check `Citus support <https://github.com/zalando/patroni/blob/master/docs/citus.rst>`__ page for more information.
+**Note to Citus users**: Starting from 3.0 Patroni nicely integrates with the `Citus <https://github.com/citusdata/citus>`__ database extension to Postgres. Please check `Citus support chapter <https://github.com/zalando/patroni/blob/master/docs/citus.rst>`__ in the Patroni documentation for more info about how to use Patroni high availability together with a Citus distributed cluster.
```
Comment (clairegiordano) [inline README.rst:17]: 1. Missing a "the" immediately before "Citus support"
2. is it OK to call it a "Citus support chapter" given that it's a single page, do you consider the page a "chapter". I'm not familiar with any terminology best practices you have for the Patroni documentation...

File: docs/citus.rst [hunk 2]
```diff
@@ -51,8 +52,10 @@ patronictl
 ----------
 
 Coordinator and worker clusters are physically different PostgreSQL/Patroni
-clusters that are just logically groupped together using Citus. Therefore in
-most cases it is not possible to manage them as a single entity.
+clusters that are just logically groupped together using
+`Citus <https://github.com/citusdata/citus>`__ database extension to
```
Comment (clairegiordano) [inline docs/citus.rst:56]: 1. Missing a "the" immediately before "Citus database extension"

File: docs/index.rst [hunk 3]
```diff
@@ -12,7 +12,7 @@ We call Patroni a "template" because it is far from being a one-size-fits-all or
 
 Currently supported PostgreSQL versions: 9.3 to 15.
 
-**Note to Citus users**: Starting from 3.0 Patroni nicely integrates with `Citus <https://www.citusdata.com>`__. Please check :ref:`Citus support <citus>` page for more information.
+**Note to Citus users**: Starting from 3.0 Patroni nicely integrates with the `Citus <https://github.com/citusdata/citus>`__ database extension to Postgres. Please check :ref:`Citus support chapter <citus>` in the Patroni documentation for more info about how to use Patroni high availability together with a Citus distributed cluster.
```
Comment (clairegiordano) [inline docs/index.rst:15]: 1. Missing a "the" immediately before "Citus support"
2. is it OK to call it a "Citus support chapter" given that it's a single page, do you consider the page a "chapter"? I'm not familiar with any terminology best practices you have for the Patroni documentation...

**Comment by hughcapet** [comment]
👍

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**